README.md

```diff
@@ -1,150 +1,25 @@
-# Digital Humanities Final
-
-![Title Image](assets/images/intro-image.png)
-
-## Instructor and Author
-
-[David J. Thomas](mailto:dave.a.base@gmail.com), [thePortus.net](http://thePortus.net)<br />
-Instructor of Ancient History and Digital Humanities<br />
-Department of History<br />
-[University of South Florida](https://github.com/usf-portal)
-
----
-
-**[View the template on a live site here](https://theportus.github.io/hacking-historical-texts-final)**
-
----
-
-## Project Template
-
-This is a starter template for final projects. When you have completed your final project, you should replace this message (README.md) with a short 1-2 paragraph description of your project.
-
-See the [Course Workbook Project Page](https://theportus.github.io/hacking-historical-texts) for more information on the final project.
-
-**REQUIREMENTS BEFORE STARTING**
-+ [GitHub account](https://github.com) created
-+ [Atom](https://atom.io) editor installed
-
----
-
-## Getting a Copy and Renaming the Repository
-
-To get started, you need to fork your own copy of this repository
-
-1. Make sure you are on the homepage for [this repo](https://github.com/usf-portal/hacking-historical-texts-final)
-2. Click the 'Fork' button, near the top right of the page, to the right of the repository title.
-3. That's it, now you have your own copy. If you notice the title of the page, you are now at *your* version of the repo located at *your* account.
-4. If you can't find it, your repo should be at `https://github.com/your-username/hacking-historical-texts-final`
-5. You should change the name of the repository, to do this...
-    1. Go to your repository homepage
-    2. Click the `Settings` tab
-    3. Under `Repository Name` put your project name. GitHub general ettiquite is to use lower case and name `your-repository-like-this`
-
----
-
-## Launching the Site
-
-To create the website from this repository, follow these steps...
-
-1. Go to your repository homepage
-2. Click the `Settings` tab
-3. Change the name of the repository to represent your project
-4. Scroll down to the `GitHub Pages` section
-5. Under `Source` click the dropdown menu and choose `master branch`
-
-**See your site at your-username.github.io/your-repo-name**
-
-*(May take a few minutes)*
+[Home](https://michaelrappa.github.io/usa-migration/) |
```

Comment:
Overall, great job on this README page (and by extension, the introduction). And, your visuals work!

One suggestion I might have (and I did not say this ahead of time so it's not something held against you), is that you can add a link on the README that points to the live website. That way, if someone finds the repo on GitHub, it will render your README file and show them a nice link they can follow to see the actual website. You already have the navigation bar which will give links, but that may not be clear to the viewer.
```suggestion
### [Click Here to See the Website!](https://Michaelrappa.github.io/usa-migration)

[Home](https://michaelrappa.github.io/usa-migration/) |
```